app/Model/TaskDuplicationModel.php

```diff
@@ -60,6 +60,9 @@ public function duplicate($task_id)
             $this->taskLinkModel->create($new_task_id, $task_id, 4);
         }
 
+        $hook_values = [ 'source_task_id' => $task_id, 'target_task_id' => $new_task_id];
```

Comment:
Other parts of the code use the term "destination" instead of "target".

Reply:
changed